.gitignore

```diff
@@ -166,7 +166,7 @@ cython_debug/
 # pixi environments
 .pixi
 *.egg-info
-src/packagenamepy/_version.py
+src/examplepyapp/_version.py
```

Comment:
The mismatch between package name and module name is a deliberate design: not all libraries follow the exact convention where the package name and the main module name matches.
A very good example would be scikit-image, and another really good example would be pytorch (the package name is pytorch, but the module name is torch).